### PR TITLE
[MISC] Alphabet: Move some views to alphabet views - Part 2

### PR DESCRIPTION
**Pull request by Irallia**
Second part of Resolving
 https://github.com/seqan/product_backlog/issues/337 and
 https://github.com/seqan/product_backlog/issues/338 and
 https://github.com/seqan/product_backlog/issues/339 and
 https://github.com/seqan/product_backlog/issues/340 and
 https://github.com/seqan/product_backlog/issues/341 and
 https://github.com/seqan/product_backlog/issues/342.

This PR applys the patches and moves the headers (snippets & test coming in third part pr)
- Move seqan::views::translate to seqan3/alphabet/views/
- Move seqan::views::translate_single to seqan3/alphabet/views/
- Move seqan::views::translate_join to seqan3/alphabet/views/

### CODE REVIEW

File: include/seqan3/range/views/translate.hpp [hunk 1]
```diff
@@ -6,796 +6,14 @@
 // -----------------------------------------------------------------------------------------------------
 
 /*!\file
+ * \brief [DEPRECATED] Provides seqan3::views::translate and seqan3::views::translate_single.
  * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
```
Comment (MitraDarja) [inline include/seqan3/range/views/translate.hpp:10]: I don't know if it is important, but usually we have author and then brief. Could you fix this?

File: include/seqan3/range/views/translate_join.hpp [hunk 2]
```diff
@@ -6,383 +6,14 @@
 // -----------------------------------------------------------------------------------------------------
 
 /*!\file
+ * \brief [DEPRECATED] Provides seqan3::views::translate_join.
  * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
```
Comment (MitraDarja) [inline include/seqan3/range/views/translate_join.hpp:10]: Here too.

**Review by MitraDarja — APPROVED** [review]
LGTM, just a minor thing.:)

File: include/seqan3/range/views/translate.hpp (hunk 1 above)
Comment (Irallia) [reply include/seqan3/range/views/translate.hpp:10]: If I understand correctly, it's the other way around. So I will correct it with the other files.
https://github.com/seqan/seqan3/wiki/Writing-Documentation#annotating-general

**Review by marehr — APPROVED** [review]

**Pull request merged**